Core/Graphics/Include/Shared/Babylon/Graphics.h

```diff
@@ -15,8 +15,10 @@ namespace Babylon
         ~Graphics();
 
         static std::unique_ptr<Graphics> CreateGraphics(const GraphicsConfiguration& config);
+        static std::unique_ptr<Graphics> CreateGraphics(const ContextConfiguration& config);
 
         void UpdateWindow(const GraphicsConfiguration& config);
+        void UpdateContext(const ContextConfiguration& config);
```

Comment:
If you call `CreateGraphics` with a `GraphicsConfiguration` and then call `UpdateContext`, does that work? Or does bgfx blow up?

Reply:
Actually, it doesn't make sense to have `UpdateContext` as a public method. Once bgfx is created with a context, you can't change it. You can change the output window but all GPU resources are linked with a context. Moving `UpdateContext`  to private.